build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'com.carrotsearch:junit-benchmarks:0.7.2'
     testCompile 'com.h2database:h2:1.3.173'
+    testCompile 'org.apache.httpcomponents:httpclient:4.5.2'
 
     testCompile 'com.github.crate.crate:crate-client:6e0d793'
```

Comment:
pls use `project.tag` instead of the hash --> so it uses always the same client version as the given server